utils/config-gen/src/components/destinationsCatalogue/destinationsConfigure/index.tsx

```diff
@@ -12,38 +12,53 @@ import theme from '@css/theme';
 import { Link } from 'react-router-dom';
 
 const markdown = {
-  GA: raw('./GA.md'),
```

Comment:
**[Not Very Important]** If we are only using v1 versions for some .md files, do we need versioning here. Can we remove the unused files and keep all the .md files in the same folder?